DuckDuckGo/AutofillLoginDetailsViewController.swift

```diff
@@ -266,12 +266,11 @@ class AutofillLoginDetailsViewController: UIViewController {
     
     @objc private func save() {
         viewModel.save()
-        delegate?.autofillLoginDetailsViewControllerDidSave(self)
```

Comment:
One note here - I've moved this to `func autofillLoginDetailsViewModelDidSave()` as otherwise it is called regardless of whether or not the save was actually successful

Reply:
I don't think so since if a login failed to save it shouldn't have created / modified the account in the secure vault, so the refreshing of the login list from secure vault would have been fine, but I can't guarantee there weren't edge cases